stl/inc/xlocnum

```diff
@@ -1461,8 +1461,8 @@ private:
         return _Fmt;
     }
 
-    _OutIt __CLRCALL_OR_CDECL _Fput(_OutIt _Dest, ios_base& _Iosbase, _Elem _Fill, const char* _Buf,
-        size_t _Count) const { // put formatted floating-point to _Dest
+    _OutIt __CLRCALL_OR_CDECL _Fput(_OutIt _Dest, ios_base& _Iosbase, _Elem _Fill, const char* _Buf, size_t _Count,

```

Comment:
I'm afraid that the current approach may cause ABI incompatibility since this function is DLL-exported.

Perhaps we should add a new function template with the `template <int = 0>` trick to avoid ABI-breaking.

Reply:
Sure, we can rename the method or add `template <int = 0>` if needed but if I understand correctly these 2 methods will be named differently: https://godbolt.org/z/nqz4xP5e6

Reply:
I'll need to look at this in detail, but I suspect that @frederick-vs-ja is correct. Note that we (currently) cannot change the DLL export surface in *any* way, including adding differently-mangled functions. If this appears on the export surface (as is likely due to the `__CLRCALL_OR_CDECL` calling convention), then we likely need to leave it alone, and add a `template <int = 0>` overload that won't be separately compiled.

Reply:
Note that you can use `dumpbin /exports` to manually check (we still need to migrate this part of the build validation from the MS-internal build, sorry).

Reply:
Yep, this affects the name mangling of the exports. I'll validate and push changes.